cmd/cost/cost.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/civo/civogo"
+	"github.com/civo/cli/config"
+	"github.com/civo/cli/utility"
+	"github.com/spf13/cobra"
+)
+
+var currentcost = &cobra.Command{
+	Use:     "cost",
+	Aliases: []string{"show"},
+	Short:   "Show the current cost",
+	Example: "civo cost",
+
+	Run: func(cmd *cobra.Command, args []string) {
+
+		// Ensure the user has set a region
+		utility.EnsureCurrentRegion()
+
+		// Create a new client to the Civo API using the user's API key
+		client, err := config.CivoAPIClient()
+		// If the user has set a region, use that
+		if common.RegionSet != "" {
+			client.Region = common.RegionSet
+		}
+
+		// If there is an error, display it and exit
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+
+		// client is already defined, so we can just call the GetCost() method
+		cost, err := client.GetCost()
```

Comment:
Does this function even exists in the SDK??
I couldn’t find it in the docs. and ive also checked https://www.civo.com/api i couldn’t find any endpoint which satisfies the current requirements.